diff --git a/Modules/Panels/Launcher/LauncherListDelegate.qml b/Modules/Panels/Launcher/LauncherListDelegate.qml
index 69e3392..5d8a2f2 100644
--- a/Modules/Panels/Launcher/LauncherListDelegate.qml
+++ b/Modules/Panels/Launcher/LauncherListDelegate.qml
@@ -18,7 +18,7 @@ NBox {
   width: ListView.view.width
   implicitHeight: launcher.entryHeight
   clip: true
-  color: entry.isSelected ? Color.mHover : Color.mSurfaceVariant
+  color: entry.isSelected ? Color.mHover : "transparent"
   forceOpaque: entry.isSelected
 
   // Prepare item when it becomes visible (e.g., decode images)